rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/core/RMCore.java

```diff
@@ -442,6 +456,17 @@ public void run() {
         }
     }
 
+    protected void signalRMCoreIsInitialized() {
+        rmCoreInitializedLock.lock();
```

Comment:
I'm afraid a deadlock can occur with this pattern. waitForRMCoreToBeInitialized() can hold the lock and await a notification, while the initActivity will never be able to reserve the lock to send the notification

Reply:
Seems CountDownLatch is preferable